browser/ui/content_settings/brave_widevine_blocked_image_model_browsertest.cc

```diff
@@ -151,32 +148,29 @@ IN_PROC_BROWSER_TEST_F(BraveWidevineBlockedImageModelBrowserTest,
 
   auto model = CreateModel();
 
-  Profile* profile = browser()->profile();
   WebContents* web_contents =
       browser()->tab_strip_model()->GetActiveWebContents();
-  std::unique_ptr<BraveWidevineContentSettingPluginBubbleModel> bubble(
-      (BraveWidevineContentSettingPluginBubbleModel*)
+  std::unique_ptr<ContentSettingBubbleModel> bubble(
           model->CreateBubbleModel(
-              browser()->content_setting_bubble_model_delegate(), web_contents,
-              profile));
+              browser()->content_setting_bubble_model_delegate(), web_contents));
 
   ASSERT_FALSE(model->is_visible());
 }
 
 #if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-class BraveWidevineIconVisibilityBrowserTest : public InProcessBrowserTest {
+class BraveWidevineIconVisibilityBrowserTest : public CertVerifierBrowserTest {
  public:
   BraveWidevineIconVisibilityBrowserTest()
       : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {}
 
   void SetUpOnMainThread() override {
-    InProcessBrowserTest::SetUpOnMainThread();
+    CertVerifierBrowserTest::SetUpOnMainThread();
     host_resolver()->AddRule("*", "127.0.0.1");
-
     // Chromium allows the API under test only on HTTPS domains.
     base::FilePath test_data_dir;
-    base::PathService::Get(brave::DIR_TEST_DATA, &test_data_dir);
```

Comment:
Since it was meant to be the chrome test dir, we just use that constant. This avoid confusion about where the files are located for this test as it's meant to be `chrome/test/data`